proxy/http2/Http2ConnectionState.cc

```diff
@@ -1067,16 +1067,33 @@ Http2ConnectionState::find_stream(Http2StreamId id) const
 void
 Http2ConnectionState::restart_streams()
 {
-  Http2Stream *s = stream_list.head;
-
-  while (s) {
-    Http2Stream *next = static_cast<Http2Stream *>(s->link.next);
+  static int x = 0;
+
```

Comment:
Shouldn't X be set to rand() or at least time()?  As it stands,  the loop after "Change the start point randomly" will always end up starting at head->next, right?